src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@patternfly/react-core';
```

Comment:
This file is named like `ActionButton.tsx` and it exports two components:
1) ActionButton
2) ActionHeader

Which one is considered to be the default one that should be commonly used? Maybe creating new file for each component would be better? Or removing `export` from `ActionButton` and renaming this file to `ActionHeader.tsx`? Let's discuss it.